### DISCUSSION TITLE
Configurable construction line separation length

### DISCUSSION THREAD

**Discussion by SiboVG** [hlorus/CAD_Sketcher#358] [Feature Requests]
To my taste, the dotted line of the construction is a bit too compact and hence looks too much like a normal sketch line. It would be great if you could change the separation length (and line section length) in the addon settings. I would also increase the default separation length, but that's personal taste.

**Comment by hlorus** [comment]
A preference option would obviously be great however we should first fix the dotted shader to be completely independent of viewport zoom and entity size.
